### DISCUSSION TITLE
Would it be possible to log where from, or what, caused a reboot?

### DISCUSSION THREAD

**Discussion by ghost** [umbraco/Umbraco-CMS#14951] [Features and ideas]
#cloud
Regardless of hot or cold reboot, would it be possible even if not 100% coverage of reasons, to make the log spit out from where a (re)boot originated?
In the Cloud Portal, you're naturally logged in, so who could be logged when manually rebooting from there, yes?
When Azure auto redistribute a site because of high load or whatever, is that detectable?
Fill in something I'm not thinking of right now: __________.
Adjusting existing log msg(s) to more clear say if it's a cold or hot reboot, also.

**Comment by sajumb** [comment]
Hi @SOTFB,
Thank you for your feature request; it offers valuable insights. We find the idea of capturing the source of reboots, whether hot or cold, to be feasible and useful for better system traceability. Adding user IDs for manual reboots initiated from the Cloud Portal is another excellent suggestion for accountability.

For Azure's automated redistributions, we believe these events can be detected and logged as well, enhancing our system's transparency. Additionally, we'll explore the possibility of adjusting existing log messages to clearly indicate whether a reboot is cold or hot.

Your suggestions will be reviewed for their feasibility and potential inclusion in our upcoming development cycles.
